.github/workflows/merge-commits.yml

```diff
@@ -140,15 +140,15 @@ jobs:
             const token = `${{ secrets.GITHUB_TOKEN }}`;
             const sha = 'main-next-${{needs.dequeue.outputs.SHA}}';
             const author = '${{needs.dequeue.outputs.AUTHOR}}';
-            const baseBranch = String(${{ env.TARGET_BRANCH }});
+            const baseBranch = '${{ env.TARGET_BRANCH }}';
             const octokit = new Octokit({ auth: token })
             const newPr = await octokit.request('POST /repos/{owner}/{repo}/pulls', {
               owner: 'microsoft',
               repo: 'FluidFramework',
               title: 'Main Next Integrate',
               body: 'This commit is queued for getting merged with the next branch. Please make sure to resolve any conflicts/CI failures seen on this PR. Thank you!',
               head: sha,
-              base: baseBranch
+              base: 'next'
```

Comment:
Did you intend to make both this change and the one above that sets baseBranch to the env variable?

Reply:
Removing the env variable for now and setting it to next directly.